api-client/src/endpoints/createPOIsEndpoint.ts

```diff
@@ -15,39 +15,36 @@ type ParamsType = {
 export default (baseUrl: string): Endpoint<ParamsType, Array<PoiModel>> =>
   new EndpointBuilder<ParamsType, Array<PoiModel>>(POIS_ENDPOINT_NAME)
     .withParamsToUrlMapper(
-      (params: ParamsType): string => `${baseUrl}/${params.city}/${params.language}/wp-json/extensions/v3/locations`
+      (params: ParamsType): string =>
+        `${baseUrl}/${params.city}/${params.language}/wp-json/extensions/v3/locations?on_map=1`
```

Comment:
what does the param stand for `on_map` ?

Reply:
only pois that should be shown on the map (we don't need anything else). Only for these pois it is guaranteed that long/lat are not null.